src/t8_geometry/t8_geometry_implementations/t8_geometry_cad.cxx

```diff
@@ -190,8 +195,17 @@ t8_geometry_cad::t8_geom_evaluate_cad_tri (t8_cmesh_t cmesh, t8_gloidx_t gtreeid
    * 
    */
 
-  /* Linear mapping from ref_coords to out_coords */
-  t8_geom_compute_linear_geometry (active_tree_class, active_tree_vertices, ref_coords, 1, out_coords);
+  printf ("ref_coords: [%f, %f, %f, %f, %f, %f]\n", ref_coords[0], ref_coords[1], ref_coords[2], ref_coords[3],
```

Comment:
What are these `printf`s doing here? Remnants from development? Forgot to delete this?